pkg/ccl/changefeedccl/sink_webhook.go

```diff
@@ -124,6 +126,21 @@ func encodePayloadWebhook(messages []messagePayload) (encodedPayload, error) {
 	return result, err
 }
 
+func encodePayloadCSVWebhook(messages []messagePayload) (encodedPayload, error) {
+	result := encodedPayload{
+		emitTime: timeutil.Now(),
+	}
+
+	var mergedMsgs []byte
+	for _, m := range messages {
+		result.alloc.Merge(&m.alloc)
+		mergedMsgs = append(mergedMsgs, m.val...)
```

Comment:
looks like we are not updating some of the fields in the result -- such as mvcc timestamp.